solana.js

```diff
@@ -0,0 +1,52 @@
+const {
+    Connection,
+    Keypair,
+    PublicKey,
+    clusterApiUrl,
+} = require("@solana/web3.js");
+const { getOrCreateAssociatedTokenAccount, transfer } = require("@solana/spl-token");
+const bs58 = require("bs58");
```

Comment:
What is this import for?

Reply:
If we plan to expand the repo further, I think we can setup a simple Node.js project with `package.json` and ESLint so it can check out for these.

Reply:
i removed the bs58 import. i initially had more helper method and that's why I had it. I agree with Quang that it could be useful if we want to expand this further!